razzle.config.js

```diff
@@ -0,0 +1,21 @@
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+module.exports = {
+  modify(config, { target, dev }, webpack) {
+    const appConfig = config; // stay immutable here
+
+    if (target === 'node' && dev) {
```

Comment:
Interestingly, when I change this to `target === 'web'`, I get an error `Error: Cannot find module "/Users/heidas03/dev/articles-renderer/build/assets.json"`. It makes sense since there is no `assets.json` created on an `npm run dev`.  I'll have to read up a bit more on this.